simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch

```diff
@@ -0,0 +1,31 @@
+<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->                                                                                                            
```

Comment:
Why/how was the CW sample app working without this previous? Always 0,0,0?

Reply:
CW sample app links to the `turtlebot3_navigation.launch` in each sub-packages, like https://github.com/aws-robotics/aws-robomaker-bookstore-world/blob/ros1/launch/turtlebot3_navigation.launch

We discussed this during backlog grooming and agreed we should put this inside `cloudwatch_simulation` because the worlds packages should be a purely world package and should not contain code for navigation.

Reply:
Thanks for the context.